lib/auth/auth.go

```diff
@@ -591,6 +592,30 @@ func (s *AuthServer) generateUserCert(req certRequest) (*certs, error) {
 	return &certs{ssh: sshCert, tls: tlsCert}, nil
 }
 
+// CheckKubeCluster validates kubernetes cluster name against known kubernetes
+// clusters. If kc is empty, this validation succeeds (no error returned).
+func CheckKubeCluster(kc string, pg services.ProxyGetter) (string, error) {
```

Comment:
Looks like this function always either returns an error, or an unmodified `kc`.  Why bother returning the string value at all?

Reply:
Removed, it was an artifact of older implementation